docs/components/grapesjs.rst

```diff
@@ -0,0 +1,74 @@
+GrapesJS Builder
+################
+
+Currently, Mautic uses the GrapesJS Builder for Emails and Landing pages. 
+
+Setup 
+*****
```

Comment:
I think we need to be a bit more verbose here. Where exactly do they run `npm install`? Do they have to have Mautic installed first? 

We need to assume that people reading this are completely new to using the builder and may not know where to start.